frontend/components/app.js

```diff
@@ -46,48 +27,75 @@ class App extends React.Component {
   }
 
   render() {
-    const { children } = this.props;
-    const storeState = this.state;
+    const { user, children, notifications } = this.props;
 
-    if (storeState.user.isLoading) {
+    if (!user || user.isLoading) {
```

Comment:
oof, good catch. I think this is another good case for https://github.com/18F/federalist/issues/958